src/views/bonds/BondsForm.js

```diff
@@ -243,6 +243,7 @@ function BondsForm(props) {
             name="coupon_rate"
             placeholder="Coupon rate"
             type="number"
+            step="0.01"
```

Comment:
We would like to have upto 3 decimal places
```suggestion
            step="0.001"
```

Reply:
Added. You didn't mention the decimal places earlier.